### ISSUE TITLE
UI - Progress widget

### ISSUE THREAD

**Issue by tristanMatthias** [code-mentoring/learn#25]
![image](https://user-images.githubusercontent.com/2550138/79711090-e95a4780-8294-11ea-9551-e6577ff87e55.png)

**Is your feature request related to a problem? Please describe.**
As someone learning how to code, I want to see how far I am generally across the languages/technologies I know.

**Describe the solution you'd like**
There should be a widget on the dashboard that displays all paths and the progress the user is through them.

**Additional context**
May want to use a canvas to draw the arc.
https://www.figma.com/file/MfOc0a8Ac9Zoh9RppoEuiD/CM-UI?node-id=2%3A0

**Data**
This will need dummy data for now. It should follow the format of:
```ts
interface PathProgress {
  path: Path // Still being worked on by @raysercast1 
  progress: number; // 0 - 1
}

// Dummy interface until it's available from #5 
interface Path {
  id: string;
  name: string;
  icon: string;
  description: string;
}

<ProgressWidget /> // fetches its own data (mocked for now)
```

**Comment by ahmedfaaid** [comment]
Hi Tristan, I'd like to take a stab at this.

Should I create dummy data for this?

**Comment by tristanMatthias** [comment]
Hey @ahmedfaaid, shame you weren't part of the UI call we just had.

Yeah that's a good quetion. I'll update the specs for what properties it should query for as dummy data.

**Comment by tristanMatthias** [comment]
Hey @ahmedfaaid , how you doing with this? (Just cleaning up issues)a

**Comment by ahmedfaaid** [comment]
Hi @tristanMatthias . So far mostly a static widget. My question is what kind of data can I seed into my local postgres database to help with more dynamic data?

**Comment by tristanMatthias** [comment]
Sorry for late response @ahmedfaaid, just saw this now. Seeders have been merged into develop, just waiting on #5 to have the paths added. Setting this as blocked till that's available.

**Comment by tristanMatthias** [comment]
Hey @ahmedfaaid, #68 was merged in now, so paths should be available for you to pull down. Have a chat with @suzynakayama to learn how to create a Container `unstated-next` for Paths in the front end.

**Comment by ahmedfaaid** [comment]
That's great! Will do so.

**Comment by ahmedfaaid** [comment]
Hey @tristanMatthias I don't think I got an invitation to the team. I can't seem to push my code.

**Comment by tristanMatthias** [comment]
Hey @ahmedfaaid, fixed now. Sorry about that!
